envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py

```diff
@@ -79,8 +87,6 @@ def _add_http_ports(agent_configuration: AgentConfiguration) -> AgentConfigurati
 depth_1_a_test_configuration = replace_agent_configuration(
     noop_test_configuration, agent_configuration
```

Comment:
Why are we using the `noop_test_configuration` as a starting point? Wouldn't it make more sense to have the most common setup already filled in, like dept: 1, local network scan: false, fingerprinters: [http], tunnel open time: 0, etc.?

Reply:
It's hard to say what the most common configuration is. The most common configuration might also change. Changes to the common configuration could have unforeseen effects on other configurations. If every configuration is built from an empty template, you don't have that concern.

In other words, if every list is empty, value is 0, boolean is false, than you simply enable/add the things you need. If you start from a common configuration, you need to add some things and remove some others. This is more prone to issues creeping in.